docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -24,7 +24,43 @@ From now on, content types missing a charset can be both parsed and expressed di
 
 *Note to scala users*: If you have `match` statements across `ContentType`, keep an eye out for new compiler hints. You need
 to decide what what to do in case you get a content type with a missing character set, by adding a 
-`ContentType.WithMissingCharset` case. 
+`ContentType.WithMissingCharset` case.
+
+### Server-Sent Events Support
+
+Support for Server-Sent events was added by merging akka-sse project by [@hseeberger](https://github.com/hseeberger). Kudos to you, Heiko!
+
+### List of Changes
+
+#### Improvements
+
+##### akka-http-core
+
+* HTTPS proxy support ([#192](https://github.com/akka/akka-http/issues/192))
```

Comment:
Do we want to mention here that we don't support authorizing to the proxy server yet?